__tests__/nc_news_test.test.js

```diff
@@ -331,4 +331,29 @@ describe('GET /api/topics', () => {
             })
         });
     });
-   
\ No newline at end of file
+    describe('DELETE /api/comments/:comment_id', () => {
+        test('Delete comment by comment ID, respond with 204 and no content', () => {
+            return request(app)
+            .delete("/api/comments/1")
+            .expect(204)
+            .then((response)=> {
+                expect(response.body).toEqual({})
+            })
+        });
+        test('Delete comment by comment ID, respond with 404 and error message when resource to be deleted is non-existent', () => {
+            return request(app)
+            .delete("/api/comments/555")
+            .expect(404)
+            .then((response)=> {
+               expect(response.body.msg).toBe("No comment found under ID:555")
+            })
```

Comment:
Ideally, we want our error messages to be generic. Being overly specific error messages gives others an idea of how to access specific information in your database 🔍